packages/kit/package.json

```
@@ -6,7 +6,7 @@
 		"@sveltejs/vite-plugin-svelte": "^1.0.0-next.10",
 		"cheap-watch": "^1.0.3",
 		"sade": "^1.7.4",
-		"vite": "2.3.3"
+		"vite": "^2.3.5"
```

I think we want to pin versions for now

```suggestion
		"vite": "2.3.5"
```